src/Unit_cell/hubbard_orbitals_info.hpp

```diff
@@ -0,0 +1,271 @@
+#ifndef __HUBBARD_ORBITAL_INFO_HPP_
+#define __HUBBARD_ORBITAL_INFO_HPP_
+
+namespace sirius {
+// structure containing all informations about a specific hubbard orbital (including the index of the radial function)
+    class hubbard_orbital_info_ {
```

Comment:
this is probably a "hubbard_orbital_descriptor"

Reply:
True. I changed the name of the class to hubbard_orbital_descriptor_.